src/baudot.c

```diff
@@ -265,7 +265,7 @@ baudot_encode( unsigned int *databits_outp, char char_out )
 
     unsigned char charset_mask = baudot_encode_table[ind][1];
 
-    debug_log("I: (baudot_charset==%u)   input character '%c' 0x%02x charset_mask=%u\n", baudot_charset, char_out, char_out, charset_mask);
```

Comment:
Actually, baudot_charset should just be changed to 'unsigned int', so I'll do that and drop this patch.